acceptance/suites/tests/555-puppetserver-service/reboot.rb

```diff
@@ -0,0 +1,55 @@
+if (options[:type] == 'pe') then
+  pssn='pe-puppetserver'
+else
+  pssn='puppetserver'
+end
+
+test_name "Validates the ability of the #{pssn} service to reliably start after a reboot."
+
+max=3
+sleeptime=45
+retryCounter=0
```

Comment:
Sure, but until then, do we have a sense of the overall wall-clock impact in the best / average (expected) / worst case?  If so, what are those cases?

Might be worth a comment in the code to this effect.  "This test isn't expected to take longer than X, but if it does the plan is to ..."